app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, old_file, path, *_ = command.split()
+
+    if command_name == "mv" and old_file != path:
+        last_slash_index = path.rfind("/")
+        path_dirs = path[:last_slash_index]
+        os.makedirs(path_dirs, exist_ok=True)
+        with open(old_file, "r") as file_out, open(path, "w") as file_in:
+            content = file_out.read()
+            file_in.write(content)
```

Comment:
There is no need to allocate a `content` variable, you could omit it
```suggestion
            file_in.write(file_out.read())
```